udf/bool_to_bytes.sql

```diff
@@ -0,0 +1,12 @@
+/* Convert a boolean to byte array */
+
+CREATE TEMP FUNCTION
+  udf_bool_to_bytes(val BOOLEAN) AS (
```

Comment:
This isn't an appropriate name for this function, I believe it's misleading given that it's in a shared namespace, especially with the description `Convert a boolean to byte array`

Reply:
```suggestion
  udf_bool_to_365bits(val BOOLEAN) AS (
```